src/rpc/mempool.cpp

```diff
@@ -910,6 +910,41 @@ static RPCHelpMan submitpackage()
     };
 }
 
+static RPCHelpMan getprioritisationmap()
+{
+    return RPCHelpMan{"getprioritisationmap",
+        "Returns a map of all user-created (see prioritisetransaction) fee deltas by txid, and whether the tx is present in mempool.",
+        {},
+        RPCResult{
+            RPCResult::Type::OBJ_DYN, "prioritisation-map", "prioritisation keyed by txid",
+            {
+                {RPCResult::Type::OBJ, "txid", "transaction txid", {
+                    {RPCResult::Type::STR_AMOUNT, "fee-delta", "transaction fee delta in " + CURRENCY_UNIT},
+                    {RPCResult::Type::BOOL, "in-mempool", "whether this transaction is currently in mempool"},
```

Comment:
These should be `"fee_delta"` and `"in_mempool"` -- hyphens are annoying because they often get interpreted as a subtraction, so it's easier to say `jq .[].fee_delta` than `jq .[]."fee-delta"`. Also more consistent with other code (cf #24528).

Reply:
Fixed